WalletWasabi/Announcements/ReleaseHighlights.md

```diff
@@ -1,49 +1,38 @@
 
 ## Release Summary
-Wasabi Wallet v2.4.0 brings send to Silent Payment addresses!
+Wasabi Wallet v2.5.0 continues to enhance Wasabi's resilience and improve the user interface to reduce frustration.
 
-### 🤫 Send to Silent Payment addresses
+### 💪 3rd Party Providers for Fee & Exchange Rate
 
-Silent Payments are privately reusable static addresses. While sending Bitcoin to them is as easy as sending to regular addresses, it is not possible for an observer to link several payments to the same Silent Payments address.
+Fee rate estimations and exchange rate providers are now configurable.
 
-They are useful to send an address to multiple persons at the same time (tips, donations...) or to make multiple payments between the same entities (salary, rent...).
+Previously, this information was provided by Wasabi's backend. This change aligns with our long-term goal of completely removing dependence on the backend.
 
-Wasabi is now one of the first wallet to support send to Silent Payment addresses, and we hope that more wallets will join! We are working on allowing receiving, please bear with us!
+After this release, Wasabi will be significantly closer to achieving that goal, with most basic wallet features functioning without an active backend connection. You can read more about the remaining steps [here](https://github.com/orgs/WalletWasabi/discussions/13661).
 
-### 🎁 Donation Button
+### 👨‍🔧 Quality of Life Features
 
-Wasabi is Free Open Source Software (FOSS).
-No service is provided, and therefore it doesn't generate any revenue.
+In the last version, we introduced a donation button to gauge community interest in funding Wasabi's future. We have been overwhelmed by the support and interest shown by our community, including long-time users and privacy enthusiasts.
 
-However, maintaining and improving the software requires ongoing effort, such as maintenance, continuous improvement, bug fixes, testing, documentation, regular updates etc...
+We want to thank you for believing in Wasabi and for your support, whether through using the software, providing feedback, contributing financially, or contributing to the open-source project in any form.
 
-While Wasabi is one of the best tools available to use Bitcoin privately, achieving flawless financial-sovereign privacy is still a work in progress.
-There are many features yet to be implemented, and your support can help make them a reality.
+To show our gratitude, we've designed this release for you, our community. We've finally addressed some of the oldest bugs and feature requests that have been long-awaited!
 
-Additionally, contributors cover static costs, such as server expenses and release management, using their personal funds.
+Listing all the improvements would be too lengthy, but here are some examples:
+- Randomly skipping rounds was removed
+- Coinjoin settings & profiles are now all in a single tab
+- More information in the status icon
+- Resync button now available in Tools
+Etc...
```

Comment:
![image](https://github.com/user-attachments/assets/88db0875-cf19-482f-bc1c-a77de5a8507b)

I don't think the **Etc** is desired like this?

Reply:
I´m missing the line break. Rider is very annoying when working with MD